lib/install.ps1

```diff
@@ -9,7 +9,7 @@ function nightly_version($date, $quiet = $false) {
     "nightly-$date_str"
 }
 
-function install_app($app, $architecture, $global, $suggested, $use_cache = $true, $check_hash = $true) {
+function install_app($app, $architecture, $global, $suggested, $use_cache = $true, $check_hash = $true, $useinnoextract) {
```

Comment:
It will be patched in manifests itself.

It is nonsense to have additional switch, and 20 checks for this.

Remove

Reply:
Sorry, I did not understand the part about the patch being in the manifest instead. Really sorry for the hassle caused.

Reply:
The patches in the manifest should be something similar to the instaler script in [this manifest](https://github.com/ScoopInstaller/Main/blob/42f1e0956330d19cc44f38375b18528187f89ff7/bucket/plex-server.json), right?